doc/configuration.rst

```diff
@@ -1882,9 +1882,9 @@ by including in your ``Makefile``:
 .. code-block:: Makefile
 
     html_abort_on_example_error:
-	$(SPHINXBUILD) -D abort_on_example_error=1 -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
-	@echo
-	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
+    $(SPHINXBUILD) -D abort_on_example_error=1 -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
+    @echo
+    @echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
```

Comment:
Maybe we want the old indentation?

CI error:

> /home/circleci/project/doc/configuration.rst:1882: WARNING: Lexing literal_block 'html_abort_on_example_error:\n$(SPHINXBUILD) -D abort_on_example_error=1 -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html\n@echo\n@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."' as "Makefile" resulted in an error at token: '-'. Retrying in relaxed mode.

Reply:
Just curious does the docs need to build with CircleCI or can I migrate them to GH actions (in separate PR)?

Reply:
CircleCI is better because the artifacts are immediately available and can be viewed as a live webpage with a single click

Reply:
Well not single but I see what you mean... We used to do the same until we switched to Readthedocs feature https://docs.readthedocs.io/en/stable/pullrequests.html which made it even easier...

Reply:
> Well not single but I see what you mean

With circleci-artifacts-redirector-action this link is meant to provide single-click access to the rendered HTML:

<img width="809" alt="Screenshot 2024-03-09 at 1 41 56 PM" src="https://github.com/sphinx-gallery/sphinx-gallery/assets/2365790/5ee30e87-fc43-4123-8c01-939477c98c96">